db/migrate/20151210225310_create_test_client_requests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTestClientRequests < ActiveRecord::Migration
```

Comment:
oh I see! Hmmmm I am not sure what to think about this.

From my perspective, having a generic client data object is about testing but it's also about moving toward composition over inheritance for client data. This would mean we could generate client data objects "on the fly" that have certain features rather than creating a "test client data" object that could display different client features. 

Haven't fully thought through the exact implementation, but I do think that having the ability to write an integration test for client data that uses parallel steps would be great step toward generalization. Having that ability doesn't just mean having a test client data object, but having the ability to inject those types of features into any type of client data object.